inc/ctest.h

```diff
@@ -331,6 +331,30 @@ do \
 } \
 while(0)
 
+// this macro expans to the needed _ToString and _Compare functions for an enum,
```

Comment:
>expans [](start = 14, length = 6)

"expands" (here and below)

Reply:
Done.

---
In reply to: [326827609](https://github.com/Azure/azure-ctest/pull/44#discussion_r326827609) [](ancestors = 326827609)